### ISSUE TITLE
return in builder's block causes closing tags to be skipped

### ISSUE THREAD

**Issue by leafo** [ageweke/fortitude#19]
This is kind of a bizarre case, not sure what the correct approach is. This would allow you to have some method in your widget that returns deep within html builder blocks and still have valid markup rendered.

```ruby
def some_widget_method
  dt class: "hello" do
    p "Yes please"
    return
  end
end
```

Erector: 

```html
<dt class="hello"><p>Yes please</p></dt>
```

Fortitude:

```html
<dt class="hello"><p>Yes please</p>
```

**Comment by ageweke** [comment]
That is an odd case, but I’m hoping judicious use of `ensure` will fix it. Trying that out now…

**Comment by ageweke** [comment]
[Fixed](https://github.com/ageweke/fortitude/blob/5de116a05c9ace493659c6f3962e040ad27f13e5/spec/system/tag_rendering_system_spec.rb#L95-L158), and included in 0.9.2, which I just released.
